src/main/java/org/generationcp/middleware/dao/GermplasmDAO.java

```diff
@@ -2067,4 +1932,71 @@ private void updateGroupSource(final List<Integer> gids, final Integer newGroupS
 		}
 	}
 
+
+	public List<Variable> getGermplasmAttributeVariables(final List<Integer> gids, final String programUUID) {
+		try {
```

Comment:
@cuenyad 
Maybe you missed this comment yesterday.
If you are using constants for the scalars then, you should use same constants when building the query.

Reply:
done.